### DISCUSSION TITLE
Grid Layout Problem

### DISCUSSION THREAD

**Discussion by aprijal99** [heroui-inc/heroui#1003] [Help]
In the documentation, it is said that the breakpoints work in the following windows sizes,

![image](https://user-images.githubusercontent.com/58645926/211996165-ddb30b4f-5e0c-4c32-96ca-03f3b76475b1.png)

I want to create a responsive grid layout consisting of 3 cards, my idea is when the window size is more than 650 px, the cards will take 3 columns size, and when the window size is less than 650 px, the cards will take 4 columns size,

```
<Grid
  xs={4}
  sm={3}
>
```

But I end up getting the xs size applied on the window size of less than 960 px, and the sm size on more than 960 px

**Comment by jrgarciadev** [comment]
Hey @aprijal99  sorry the delay, please upgrade to  [V2](https://nextui.org/blog/nextui-v2) and use TailwindCSS Grid System https://tailwindcss.com/docs/display#grid
